CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+
+## 1. Report a valid issue :heavy_check_mark: 
+
+BIDS-Apps uses [GitHub's issue tracker](https://github.com/BIDS-Apps/bids-apps.github.io/issues). All bugs and enhancements should be listed so that we don't lose track of them, can prioritize and assign them to the relevant developer or maintainer.
+
+Consider the following recommended best practice for writing issues, which are (Recommended but not limited to):
+1. More detailed description rather than one-liners
+2. Screenshots 
+3. Providing example files and error logs
+4. How to reproduce it
+5. Details of your local system or environment that you're using
+
+## 2. Labels :label: 
+
+Labels let us identify different types of issues, pull requests and helps in categorizing them based on their characteristics. Please take a look at the different types of labels we use.
+
+- ![Bug](https://img.shields.io/badge/-Bug-%23ee0701) *These issues points to the problems causing the project to crash, producing an invalid output or causing an error.*
+If you find a bug, feel free to open an issue with the detailed description, screenshots and how to reproduce it.
+If you experience a bug that is already listed in the issues, do comment down your remark and wait till our developers get back to you.
+
+- ![Duplicate](https://img.shields.io/badge/-duplicate-%237E7E7E) *It indicates similar/existing issues or pull requests.* 
+Do check the existing issues and pull request before creating a new one to avoid duplicacy.
+If you find a duplicate issue or PR, please give the reference in the comments.
+
+- ![Enhancement](https://img.shields.io/badge/-enhancement-%232e5783) *It indicates a new feature request or enhancement.*
+Any change or upgrade that increases the project capabilities, performance or scalability.
+In general, enhancements include:
+    - Additional functionality
+    - Error/bug repair and handling
+    - Better code coverage
+    - Improved performance
+
+- ![Help Wanted](https://img.shields.io/badge/-help%20wanted-%23128a0c) *Indicates that a maintainer or developer wants help on an issue or pull request.*
+If you have a particular skill that can help in solving the problem then consider reading through these issues as they are a great place to offer your expertise.
+Not everyone can do everything. This is the time, the community needs your help so you better dive in!
+
+- ![Invalid](https://img.shields.io/badge/-invalid-%239a9a9a) *It indicates that an issue or pull request is no longer relevant.*
+
+- ![Question](https://img.shields.io/badge/-question-%23cc317c) *It indicates that an issue or pull request needs more information.*
+These issues are some questions regarding the project that might need clarifications and a bit of discussion. 
+
+## 3. Create a pull request :zap: 
+
+Pull requests with patches, improvements and new features are a great help. Please keep them clean from unwanted commits (One Commit One PR Rule)
+
+Follow the steps to get your work included in the project.
```

Comment:
more stylistic - I think we want to nurture contributions rather than making them appear rigidly defined 

```suggestion

```